content/docs/integrations/pagerduty/index.md

```diff
@@ -10,43 +10,100 @@ integrationType: OSS plugin
 
 ## Introduction
 
+The PagerDuty Backstage plugin allows Roadie Backstage users to:
+
+ 1. See who is on call for a particular catalog Component.
+ 2. See if there are active incidents for a given catalog Component.
+ 3. Create new incidents from Backstage.
+
+![PagerDuty plugin in Roadie Backstage](./pagerduty-plugin.png)
+
 In order to use the PagerDuty plugin, Roadie needs an API token to communicate with it's APIs.
 
+This page describes how to create and set up the API token.
+
+## Steps
+
+### Step 1: Create an API token
+
+Inside PagerDuty, access "API Access Keys" inside the "Integrations" menu.
+
+![API Access menu Item on PagerDuty integrations menu](./create-api-token.png)
+
+On the API Access Keys page, click the "Create New API Key" button.
 
-These are set within backstage at the following url:
+![Create new api key button](./create-new-api-key.png)
+
+Set a sensible description and click "Create Key".
+
+![Create key modal](./create-key.png)
+
+ℹ️  The PagerDuty Backstage plugin allows users to create PagerDuty incidents from within Backstage. For this reason, the API key needs both read and write permissions to function correctly.
+
+Copy the key that is created to your clipboard.
+
+### Step 2: Store the credentials in Roadie
+
+Visit the following URL and enter the API Key value from above into `PAGERDUTY_TOKEN`.
 
 ```text
 https://<tenant-name>.roadie.so/administration/settings/secrets
 ```
 
-This page describes how to create and set up the API token.
+ℹ️  Roadie accepts the token from PagerDuty unmodified. The configuration steps for the PagerDuty plugin in self-hosted Backstage are more complex, and will not work in Roadie.
 
-## Steps
+### Step 3: Add the PagerDuty annotation
```

Comment:
Added steps to show how to set the annotation.

Reply:
Looks great, thank you.